tests/ruby.test.ts

```diff
@@ -205,4 +205,62 @@ p tukey_array
   assert.equal(code.language, 'Ruby');
 });
 
+test('ludic numbers', () => {
```

Comment:
This is not a ludic numbers problem, please rename this

Reply:
Ya sure